<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>visualizadorMusica</class>
 <widget class="QWidget" name="visualizadorMusica">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>585</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>450</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="sizeIncrement">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">.QLabel{
	font-size: 12pt;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item row="3" column="0" colspan="3">
    <widget class="QSlider" name="barraVolume">
     <property name="styleSheet">
      <string notr="true">QSlider::handle:horizontal {
	background: red;
}</string>
     </property>
     <property name="value">
      <number>99</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="0" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QScrollArea" name="areaDetalhes">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
border-style: none;</string>
     </property>
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>280</width>
        <height>225</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <property name="horizontalSpacing">
        <number>0</number>
       </property>
       <property name="verticalSpacing">
        <number>6</number>
       </property>
       <item row="5" column="0">
        <widget class="QLabel" name="faixaMusica">
         <property name="text">
          <string>Faixa : </string>
         </property>
        </widget>
       </item>
       <item row="6" column="0">
        <widget class="QLabel" name="generoMusica">
         <property name="text">
          <string>Gênero : </string>
         </property>
        </widget>
       </item>
       <item row="8" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="nomeAlbum">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Álbum : </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="anoAlbum">
         <property name="text">
          <string>Ano : </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="tituloMusica">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Música : </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nomeArtista">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Artista : </string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="7" column="0">
        <widget class="QLabel" name="classificacaoMusica">
         <property name="text">
          <string>Classificação : </string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="arteAlbum">
     <property name="minimumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>250</width>
       <height>250</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="estilos.qrc">:/imgs/np_cp_pad.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="3">
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>40</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="2">
       <widget class="QPushButton" name="botaoReproduzir">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
}

QPushButton::hover {
	background-color: rgb(255, 170, 0);
	border-style: none;
}

QPushButton::pressed {
	
	background-color: rgb(255, 170, 127);
	border-style: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="estilos.qrc">
          <normaloff>:/imgs/controles/pl.PNG</normaloff>:/imgs/controles/pl.PNG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="botaoPausar">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
}

QPushButton::hover {
	background-color: rgb(255, 170, 0);
	border-style: none;
}

QPushButton::pressed {
	
	background-color: rgb(255, 170, 127);
	border-style: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="estilos.qrc">
          <normaloff>:/imgs/controles/ps.PNG</normaloff>:/imgs/controles/ps.PNG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="botaoAnterior">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
}

QPushButton::hover {
	background-color: rgb(255, 170, 0);
	border-style: none;
}

QPushButton::pressed {
	
	background-color: rgb(255, 170, 127);
	border-style: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="estilos.qrc">
          <normaloff>:/imgs/controles/rw.PNG</normaloff>:/imgs/controles/rw.PNG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="botaoProximo">
        <property name="styleSheet">
         <string notr="true">QPushButton {
	background-color: transparent;
}

QPushButton::hover {
	background-color: rgb(255, 170, 0);
	border-style: none;
}

QPushButton::pressed {
	
	background-color: rgb(255, 170, 127);
	border-style: none;
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="estilos.qrc">
          <normaloff>:/imgs/controles/fw.PNG</normaloff>:/imgs/controles/fw.PNG</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>40</width>
          <height>40</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="estilos.qrc"/>
 </resources>
 <connections/>
</ui>
